tests/vm_test.go

```diff
@@ -1921,6 +1924,66 @@ status:
 			}, 2*time.Minute, 1*time.Second).Should(Succeed())
 		})
 	})
+
+	Context("[Serial] when node becomes unhealthy", Serial, func() {
+		const componentName = "virt-handler"
+		var nodeName string
+
+		getHandlerNodePod := func() *k8sv1.PodList {
+			pods, err := virtClient.CoreV1().Pods(flags.KubeVirtInstallNamespace).List(context.Background(),
+				metav1.ListOptions{
+					LabelSelector: fmt.Sprintf("kubevirt.io=%s", componentName),
+					FieldSelector: fmt.Sprintf("spec.nodeName=" + nodeName),
+				})
+			Expect(err).NotTo(HaveOccurred())
+			Expect(pods.Items).To(HaveLen(1))
+
+			return pods
+		}
+
+		AfterEach(func() {
+			libpod.DeleteServiceBlackHoleForKubernetesApiToPod(getHandlerNodePod(), virtClient, componentName)
+			Eventually(getHandlerNodePod().Items[0], 120*time.Second, time.Second, HaveConditionTrue(k8sv1.PodReady))
+		})
+
+		It("[Serial] the VMs running in that node should be respawned", func() {
+
+			By("Starting VM")
+			vm := startVM(virtClient, createVM(virtClient, libvmi.NewAlpine()))
```

Comment:
```suggestion
			vm := startVM(virtClient, createVM(virtClient, libvmi.NewCirros()))
```

Cirros should suffice to reduce resource usage?